### ISSUE TITLE
ST-USERFLOW Workshop Round 2 (Round 5 Workshop) PLACEHOLDER 

### ISSUE THREAD

**Issue by jenewingpierce** [raft-tech/TANF-app#666]
Problem Statement:
Supports [#506](https://github.com/raft-tech/tanf-app/issues/506) by adding content concerning the STT user journey / user flow. This issue will deliver ST workshop mural board and facilitation guide that we will use to conduct the ST workshop planned for Feb 12.

AC: 
- [x] The problem we are trying to solve is clearly stated, as are questions we still have 
- [x] Facilitation guide accompanies ST workshop mural board 

Tasks
- [ ] STT related journey map
- [x] STT related user flow
- [x] Problem framing section
- [x] Facilitation guide
- [x] Place journey map, user flow, and problem framing on mural board 

DD
- [ ] Documentation: [Facilitation guide](https://hackmd.io/Ful-2KY9TAOjgATDvBchSQ) and [Mural board](https://app.mural.co/t/raft2792/m/raft2792/1611589331046/afcaddac81d3da8ebb0a81349da3939957608807) has been shared with the PO and team for feedback
- [ ] @lfrohlich has reviewed and signed off

**Comment by amilash** [comment]
@reitermb is this work done already? If so can you assign to @lfrohlich  and make a note for her to close the ticket? 
If not let me know and we can prioritize, thanks!

**Comment by reitermb** [comment]
> @reitermb is this work done already? If so can you assign to @lfrohlich  and make a note for her to close the ticket? 
> If not let me know and we can prioritize, thanks! 

Yes, this was done/duped by other tickets from around the same time. Should be good to close @lfrohlich
